kura/distrib/src/main/resources/fedora25/README.md

```diff
@@ -0,0 +1,26 @@
+Manual steps to install Kura 3.0 on Fedora 25 / Raspberry Pi
+==
+
+1. Download Fedora Minimal image from [here](https://arm.fedoraproject.org/).
+1. Copy downloaded image to your sdcard. There are many references on how to do this for your development platform,
+such as the [Raspberry Pi installation guide](https://www.raspberrypi.org/documentation/installation/installing-images/)
+      The command for the Fedora 25 image on Linux or Mac is something like:
+      ```
+      xzcat ~/Downloads/fedora/Fedora-Minimal-armhfp-25-1.3-sda.raw.xz | sudo dd bs=1m of=/dev/rdisk2
+      ```
+1. Load the sdcard above and boot the RPi.
+1. Scan your network for the RPi (if using DHCP).
+      ```
+      sudo nmap -sP 192.168.0.0/24 | awk '/^Nmap/{ip=$NF}/B8:27:EB/{print ip}'
+      ```
+1. SSH into the RPi. You will need to be plugged into a wired ethernet network at this point,
+but after this step you could configure wifi and repeat the discovery step.
+1. Grow the root file system as described [here](https://fedoraproject.org/wiki/Architectures/ARM/F25/Installation#Resize_the_Root_Filesystem)
```

Comment:
Maybe I didn't get the point.
This is where I land when clicking the link in line 18:

![schermata 2017-01-25 alle 14 49 59](https://cloud.githubusercontent.com/assets/10448126/22292895/bea22dd4-e30d-11e6-9d2b-476665334fe9.png)

This is what I needed to complete the step via command line. It is in the subsequent FAQ section:

![schermata 2017-01-25 alle 14 50 15](https://cloud.githubusercontent.com/assets/10448126/22292952/f9c27c98-e30d-11e6-9750-452fafa083ac.png)